### PR TITLE
Fix webpack config globalObject for NextJS / SSR compatibility

### PR DESCRIPTION
**Pull request by wfortin**
This solves #52.

See https://stackoverflow.com/questions/64639839/typescript-webpack-library-generates-referenceerror-self-is-not-defined.

Tested locally in a Next JS SSR projects and it solves the issue. I would really appreciate if you could publish a new version with this fix.

### CODE REVIEW

**Comment by seawind543** [comment]
Hi @wfortin, 
Thank you for the information and also for providing the solution. I had planned to make a new release soon and would like to include the fix of #52 👍  
Therefore, I'm wondering does there any help can I provide for this pull request?

**Comment by wfortin** [comment]
https://github.com/seawind543/react-token-input/pull/53/files#diff-1fb26bc12ac780c7ad7325730ed09fc4c2c3d757c276c3dacc44bfe20faf166fR27
This is the only change that is needed if you can bundle it in the release that would be greatly appreciated.

**Review by seawind543 — CHANGES_REQUESTED** [review]
Hi @wfortin,

Apologize for the late response and inconvenience. I find out what happened now, it looks like I did not public my previous code-review comment 😭
This is my first time to make code-review via GitHub. Please let me know if I did anything wrong 🙏

File: docs/index.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 <meta name="description" content="">
 <meta name="viewport" content="width=device-width, initial-scale=1">
 <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
-<script defer src="bundle.js?99f567bcc3b9f44a03b4"></script></head>
+<script defer src="bundle.js?12c7528d60e958cc1795"></script></head>
```
[the remark this answers is not in the document]
Comment (wfortin) [reply docs/index.html]: Resolved now.

[the remark this answers is not in the document]
Comment (wfortin) [reply docs/index.html]: Fixed the merge conflicts.

**Review by seawind543 — APPROVED** [review]
Hi @wfortin, Thanks for the fix. I will release a new version soon 👍

**Pull request merged**